### ISSUE TITLE
AttributeError: 'InventoryManager' object has no attribute 'get_group'

### ISSUE THREAD

**Issue by ktdreyer** [willthames/ansible-review#67]
We have a script that calls `review()`, and this crashes in `same_variable_defined_in_competing_groups()`, because we call `get_group()` on `InventoryManager`, and this method does not exist.

```
$ python $(git rev-parse --show-toplevel)/test/run-review.py
Traceback (most recent call last):
  File "/home/kdreyer/dev/plm-playbooks/test/run-review.py", line 121, in <module>
    errors = errors + arg.review(settings, lines)
  File "/usr/lib/python2.7/site-packages/ansiblereview/__init__.py", line 76, in review
    return utils.review(self, settings, lines)
  File "/usr/lib/python2.7/site-packages/ansiblereview/utils/__init__.py", line 120, in review
    result = standard.check(candidate, settings)
  File "/usr/lib/python2.7/site-packages/ansiblereview/groupvars.py", line 76, in same_variable_defined_in_competing_groups
    group = inv.get_group(os.path.basename(candidate.path))
AttributeError: 'InventoryManager' object has no attribute 'get_group'
```

I tried downgrading ansible a few times, and it's the same result for each of these:
ansible-2.6.4-1.fc28
ansible-2.5.5-2.fc28
ansible-2.4.3.0-2.fc28

**Comment by ktdreyer** [comment]
I think I've fixed this in https://github.com/willthames/ansible-review/pull/68

**Comment by ktdreyer** [comment]
Fix is on PyPI as 1.13.8
